src/Writer/CsvWriter.php

```diff
@@ -137,6 +137,13 @@ public function write(array $data): void
             EXCEPTION);
         }
 
+        // prevent csv injection
+        $patterns = ['/^=/', '/^\+/', '/^-/', '/^@/'];
+        $replace = ['!=', '!+', '!-', '!@'];
+        foreach ($data as $key => $value) {
+            $data[$key] = preg_replace($patterns, $replace, $value);
+        }
```

Comment:
As pointed out by @VincentLanglet , that can break a lot of things. Here, we don't know that this is going to go through Excel, I don't think this writer should be changed.